endpointmanager/pkg/chplendpointquerier/chplendpointquerier.go

```diff
@@ -102,7 +123,50 @@ func QueryCHPLEndpointList(chplURL string, fileToWriteTo string) {
 		TRIARQPracticeWebscraper(chplURL, fileToWriteTo)
 	} else if chplURL == napchareURL {
 		NaphCareWebscraper(chplURL, fileToWriteTo)
+	} else if chplURL == goldblattURL {
+		BundleQuerierParser("https://fhir-test.csn.health/gs-fhir-domain-server/public-base-service-endpoints.json", fileToWriteTo)
+	} else if chplURL == cyfluentURL {
+		SwaggerUIWebscraper(chplURL, fileToWriteTo)
+	} else if chplURL == interopxURL {
+		InteropxWebscraper(chplURL, fileToWriteTo)
+	} else if chplURL == mphrxURL {
+		SwaggerUIWebscraper("https://atdevsandbox.mphrx.com/", fileToWriteTo)
```

Comment:
I found it by hovering over the Resources dropdown on the site the CHPL link brings you to and clicking on "Developer Portal". Then on the page that brings you to, I clicked on the API Documentation link and it brought me to that URL. If you don't think it should be included because I had to navigate through the website to find it, let me know!

Reply:
Yeah I say lets skip it, it looks like the URL is a dev URL and not a legit one, unfortunately